pages/understanding-json-schema/structuring.md

```diff
@@ -215,10 +215,9 @@ identifies the highlighted subschema in the following schema.
   "$id": "https://example.com/schemas/address",
   "type": "object",
   "properties": {
-    "street_address":
-       { "type": "string" },
-       "city": { "type": "string" },
-       "state": { "type": "string" }
+    "street_address": { "type": "string" },
+    "city": { "type": "string" },
+    "state": { "type": "string" }
     },
```

Comment:
This should be pulled back by two spaces.

Reply:
Done in 1ae08de.